curvefs/src/metaserver/partition.h

```diff
@@ -52,7 +52,8 @@ class Partition {
  public:
     Partition(PartitionInfo partition,
               std::shared_ptr<KVStorage> kvStorage,
-              bool startCompact = true);
+              bool startCompact = true, bool startVolumeDeallocate = true);
+    Partition() = default;
```

Comment:
Why we need a default constructed Partition?

Reply:
for unit test.